manifests/init.pp

```diff
@@ -71,6 +71,7 @@
   $mysql_connector_format                                           = 'tar.gz',
   Stdlib::Absolutepath $mysql_connector_install                     = '/opt/MySQL-connector',
   Stdlib::HTTPUrl $mysql_connector_url                              = 'https://dev.mysql.com/get/Downloads/Connector-J',
+  $mysql_connector_jar_suffix                                       = '-bin',
```

Comment:
Can you Add a datatype here as well? I guess `String[1]` works

Reply:
Done.